src/components/PromptForm/DesktopPromptForm.tsx

```diff
@@ -291,19 +304,23 @@ function DesktopPromptForm({
     // Otherwise, let the default paste handling happen
   };
 
+  const activeBorder = useColorModeValue("green.100", "green.600");
```

Comment:
```suggestion
  const activeBorder = useColorModeValue("green.200", "green.600");
```

I would change the lighter version to `green.200`. Current shade is too light.

**Before:**
![image](https://github.com/user-attachments/assets/716f044a-ccd0-4e02-97ff-89305e2d1797)

**After:**
![image](https://github.com/user-attachments/assets/383916a5-826c-43ed-8bae-fa557f9e8708)

Reply:
Thanks, I've done a version of this.  I still think it could be made better than what I can do in a follow-up, but now it works.